website/content/contributors/getting-started.md

```diff
@@ -75,11 +75,11 @@ If you're a technical contributor, there are some specific tools and technologie
 
 Even if you're not a technical contributor, there are still many ways you can contribute to open source. Here are some of the key skills you'll need:
 
-**Documentation:** Most open source projects have documentation that needs to be written and maintained. If interested in writing, you can contribute by writing documentation, tutorials, or blog posts.
+**Documentation:** Most open source projects have documentation that needs to be written and maintained. You can contribute by writing documentation, tutorials, or blog posts if you're interested in writing.
```

Comment:
Maybe leave off the "if you are interested part"?

Reply:
I think you're right.